### ISSUE TITLE
"Pipes and Filters" episode, `Sort` example.

### ISSUE THREAD

**Issue by kekoziar** [swcarpentry/shell-novice#748]
In the "Pipes and Filters" episode, the example 

## What Does `sort -n` Do?

is incomplete and does not include command lines.  Also, it is positioned before sort is actually covered in the lesson, bisecting the curriculum.

**Comment by colinmorris** [comment]
I agree that this could be improved.

I wonder if the "What does `sort -n` do?" content should just be integrated into the main body of the lesson. As an exercise, it seems pretty anemic.

**Comment by shals3** [comment]
I just created a pull request to move "What does `sort -n` do?" up a level into the main body. Also I don't think that this part is incomplete as its not really an example (hence the reason I removed it from looking like an example) but it kind of like an active learning exercise where the student look to see what is happening to figure out what the flag -n means. At the end, there is then the question before going into more detail about what sort does with working examples of nelles' data.
